arch/risc-v/src/common/espressif/Bootloader.mk

```diff
@@ -35,8 +35,14 @@ BOOTLOADER_CONFIG    = $(BOOTLOADER_SRCDIR)/bootloader.conf
 
 MCUBOOT_SRCDIR     = $(BOOTLOADER_SRCDIR)/mcuboot
 MCUBOOT_ESPDIR     = $(MCUBOOT_SRCDIR)/boot/espressif
-MCUBOOT_URL        = https://github.com/mcu-tools/mcuboot
 MCUBOOT_TOOLCHAIN  = $(TOOLSDIR)/mcuboot_toolchain_espressif.cmake
+ifndef MCUBOOT_VERSION
+	MCUBOOT_VERSION = $(CONFIG_ESPRESSIF_MCUBOOT_VERSION)
```

Comment:
```suggestion
	MCUBOOT_VERSION ?= $(CONFIG_ESPRESSIF_MCUBOOT_VERSION)
```

Reply:
I just think the `?=` syntax here is more concise and easier to understand.

Reply:
Can it be provided on a separate PR? I would need to change the original implementation (for `ESP_HAL_3RDPARTY_[VERSION|URL]` and re-test everything on our internal CI. For the sake of consistency, we should keep them equal.